deployment/helm/template/values.yaml

```diff
@@ -66,3 +69,13 @@ tolerations: []
 # autoscaling enough new nodes are brought up to leave room for the
 # plugin on each new node.
 podPriorityClassNodeCritical: true
+
+# extra ports to expose to the host
+#
+# Example
+#
+# # Note: This is not necessary for collecting metrics.
+# ports:
+#   - name: metrics
+#     container: 8891
+#     host: 8891
```

Comment:
` helm install ....--set ports[0].name=metrics   --set ports[0].container=8891   --set ports[0].host=8891`

can/should we add another example of how to pass those fields via cmd when doing helm install?

Reply:
Well we could, but I think that's the only port we can have open ATM in any of the policies.